Sprint-3/implement/rotate-char.test.js

```diff
@@ -41,3 +41,56 @@ console.log(rotateCharacter("7", 5)); // Output: "7" (unchanged, not a letter)
 // And the function should return the rotated character as a string (e.g., 'z' rotated by 3 should become 'c', 'Z' rotated by 3 should become 'C').
 console.log(rotateCharacter("z", 1)); // Output: "a" (preserves case, but wraps around)
 console.log(rotateCharacter("Y", 2)); // Output: "A" (preserves case, but wraps around)
+
+function rotateCharacter(char, shiftNum) {
```

Comment:
There's a lot of repetition in here. I wonder, could you extract a function to reduce some of the repetition? Put a note in your calendar to come back in a couple of weeks and refactor this with a helper function.

Reply:
Thank you @SallyMcGrath I put a note in my calendar. That is a great idea to come back in a few weeks